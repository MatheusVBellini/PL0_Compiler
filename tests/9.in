VAR value;
BEGIN
  IF value <> 10 THEN
    value := 10;
END.
