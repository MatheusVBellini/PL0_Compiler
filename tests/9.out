2:11 error: Missing `END` symbol before this token
2:11 error: Expected a period at the end of the program

Compilation failed, 2 errors
